### ISSUE TITLE
Faroe Islands is not in Dennmark

### ISSUE THREAD

**Issue by Fossdal** [mapillary/mapillary_issues#2131]
You have classified all cities in Faroe Islands as beeing in Dennmark

Platform:  ALL
### Steps to reproduce behavior
1. go to webpage
2. search for Torshavn
### Expected behavior

Searchresult shows Tórshavn, Faroe Islands
### Actual behavior

Search results show Tórshavn, Dennmark
### Additional information

Faroe Island is a Country but we are ruled by Dennmark

**Comment by talllguy** [comment]
Interesting! I enjoy this type of geographic orders of organization stuff. Is Faroe Islands a country ruled by Denmark like Canada is a country ruled by the Queen of England?

On the technical side, Mapillary search is powered by Mapzen, which might want to address this concern as well. [Here's a blog post about that](http://blog.mapillary.com/update/2015/10/15/powered-by-mapzen-search.html).

**Comment by talllguy** [comment]
You may want to open an issue here, and then post a link to it back on this issue: https://github.com/pelias/pelias/issues

**Comment by peterneubauer** [comment]
@Fossdal I'm coming to Torshamn next week to map some of the country with my son. Let's discuss over a coffee and see what we can do? :)

**Comment by knikel** [comment]
@Fossdal thanks for the report, I forwarded it to awesome people at Mapzen. If you'd like to read more about why it might have happened, [here's a nice overview](https://mapzen.com/blog/assult-on-copenhagen).

**Comment by Fossdal** [comment]
i opened a issue on the pelias issue tracker
https://github.com/pelias/pelias/issues/384

@talllguy yes almost same as canada and england

**Comment by talllguy** [comment]
@knikel thanks for sharing that blog post, awesome!

**Comment by Fossdal** [comment]
Think this is solved now
if i search for "Tórshavn" i get "Tórshavn, Faroe Islands"

**Comment by GITNE** [comment]
A similar problem exists for the Principality of Monaco. Although the search seems to be correct and the map seems to jump fairly closely on the territory of Monaco, all edits are labeled as being in France. The Republic of France and the Principality of Monaco are two distinct nation states, even though they share a common language and history. @mapillary Can anyone at Mapillary forward this issue to Mapzen please.

**Comment by GITNE** [comment]
While at it, Mapzen could also take a look at Réunion. The search produces “Réunion, Réunion” which is only sort of correct. Actually, it should be “Réunion, France” because although Réunion is an oversees department, it still is effectively a department of the Republic of France, hence soil and territory of France. Réunion is not a sovereign nation state (in contrast to the Faeroe Islands).

**Comment by knikel** [comment]
@GITNE: Could you provide `id` for the shape that has this mistake using https://whosonfirst.mapzen.com/spelunker? Then it's a matter of fixing it in https://github.com/whosonfirst-data/whosonfirst-data/search?p=2&q=reunion&utf8=%E2%9C%93 which is the source of truth for Mapzen's search data.

**Comment by GITNE** [comment]
> Could you provide id for the shape that has this mistake using https://whosonfirst.mapzen.com/spelunker?

@knikel Let me put it this way: The ISO has assigned the ISO 3316-1 `RE` two letter code for the *island* *of* *Réunion*, hence the reason it is currently listed as a `country`, which it isn't. Actually, it should be a `dependency` or `region`. But, for the sake of the ISO 3316-1 two letter code, Réunion `country` could remain an alias for ISO 3316-2:RE (Réunion `dependency` or `region`). So effectively, both searches `Réunion` and `Réunion, France` should point to the same id. Currently, `Réunion, France` does not exist. The search `Réunion, Réunion` is just wrong by any means.

Basically, the same applies to French Guiana. It is an overseas department and region of France, not a country although it has been assigned the ISO 3316-1 code `GF`.

Mapillary's search seems to pull data from different sources, including shops, amenities, and places from OpenStreetMap. What is the update cycle of the search index? And, why not use [Nominatim](https://nominatim.openstreetmap.org)?
